e2etest/zt_preserve_access_tier_test.go

```diff
@@ -99,3 +99,36 @@ func TestTier_V2ToClassicAccountNoPreserveCool(t *testing.T) {
 		},
 	}, EAccountType.Classic(), EAccountType.Standard(), "")
 }
+
+// Set Tier on Premium Block Blob account is a private preview feature that requires a whitelisted subscription
+// and is only available in certain regions. The service version must be 2021-04-10 or above. To test this feature,
+// uncomment TestTier_V2ToPremiumBBAccountCool and TestTier_V2ToPremiumBBAccountHot.
```

Comment:
Please create an item to remind us to reenable this test

Reply:
This test might not be reenabled since it requires a very specific subscription and region enabled.